test/1.4/errors/T_ESYNTAX_method_qualifiers.dml

```diff
@@ -0,0 +1,36 @@
+/*
+  © 2022 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.4;
+device test;
+
+// no error
+independent memoized method a() {}
+memoized independent method b() {}
+
+/// ERROR ESYNTAX
+independent independent method c() {}
+/// ERROR ESYNTAX
+memoized memoized method d() {}
+/// ERROR ESYNTAX
+memoized independent memoized method e() {}
+/// ERROR ESYNTAX
+independent memoized independent method f() {}
+
```

Comment:
Add cases for `inline independent method` and `inline memoized method` ? I expect those to fail (all nullary inline methods are banned already, but that happens post-parse so the ESYNTAX should win).

Reply:
`inline independent method` and `inline memoized method` wouldn't parse. So at the very least, they'd need their own dedicated test, and I don't see why they should if e.g. `shared inline method` doesn't have one.